website/pages/docs/secrets/transform/index.mdx

```diff
@@ -129,6 +129,52 @@ want to transform.
 - **Alphabets** provide the set of valid UTF-8 character contained within both
 the input and transformed value on FPE transformations.
 
+## Transformations
+
+### Format Preserving Encryption
+
+Format Preserving Encryption (FPE) performs cryptographically secure
+transformation via FF3-1 to encode input values while maintaining its data
+format and length.
+
+#### Tweak Source
+
+FF3-1 uses a non-confidential parameter called the tweak value along with the
+ciphertext when performing encryption and decryption operations.
+
+In order to simplify the flow of encoding and encoding operations, transformation
+creation can take care of generating and associating a tweak value. This allows
+applications to provide a single value without having the need to generate or
+store any other metadata.
+
+In cases where more granularity is required, a tweak value can be generated by
+Vault and returned, or it may be independently generated and provided.
+
+In summary, there are three ways in which the tweak value may be sourced:
+
+- `supplied`: This is the default behavior for FPE transformations. The tweak
+  value must be generated externally, and supplied into the on encode and decode
+  operations.
+- `generated`: The secret engine will take care of generating the tweak value
+  on encode operations and return this back as part of the response along
+  with the encoded value. It is up to the application to store this value
+  so that it can be provided back when decoding the encoded value.
+- `internal`: The secret engine will generate an internal tweak value per
+  transformation. This value is not returned on encode or decode operations
+  since it gets re-used for all encode and decode operations for the
+  transformation. Depending on the uniqueness of the dataset, this mode may
+  introduce higher risks, but provides the most convenience since the value does
+  not need to be stored separately.
```

Comment:
Let's use an even stronger warning here. Something like: "This mode should only be used if the values being encoded are sufficiently unique." or something of the sort.